usecases/sharding/remote_index.go

```diff
@@ -473,8 +473,10 @@ func (ri *RemoteIndex) queryAllReplicas(
 		if len(resp) == 0 {
 			return nil, errList
 		}
-		if len(resp) != len(replicas)-1 {
+		if len(resp) < len(replicas)-1 {
 			log.Warnf("full replicas search has less results than the count of replicas: have=%d want=%d", len(resp), len(replicas)-1)
+		} else if len(resp) > len(replicas)-1 {
```

Comment:
if you'd rather this be:
```suggestion
		} else if len(resp) == len(replicas) {
```
lmk

Reply:
or we could just not log at all...